cmd/kube-controller-manager/app/plugins.go

```diff
@@ -43,7 +43,6 @@ import (
 	"k8s.io/kubernetes/pkg/volume/portworx"
 	"k8s.io/kubernetes/pkg/volume/quobyte"
 	"k8s.io/kubernetes/pkg/volume/rbd"
-	"k8s.io/kubernetes/pkg/volume/scaleio"
```

Comment:
@liggitt mentioned there was a way to have the api server return a warning. Could you followup on that?

Reply:
Discussed offline. I am ok with merging this first and then following up with the warning message once it's available.

Reply:
@liggitt can you help to approve this? Thanks a lot!